src/classes/ALLO_Allocations_TEST.cls

```diff
@@ -531,7 +508,7 @@ private with sharing class ALLO_Allocations_TEST {
         setupSettings(new Allocations_Settings__c(Default_Allocations_Enabled__c = true, Default__c = defaultGau.id));
 
         //setting flags to not create default allocations for the new opportunities
-        ALLO_Allocations_TDTM.resetFlags(true);
+        TDTM_TriggerHandler.toggleTriggerState('Opportunity', 'ALLO_Allocations_TDTM', false);
```

Comment:
I feel like this is fine since this is a test context. Incrementing the counter feels artificial; this solution does feel more explicit. Maybe we can toggle it back after the DML to not encourage set-and-forget?